build-tools/scripts/RunTests.targets

```diff
@@ -18,7 +18,7 @@
     <_ApkTestProject Include="$(_TopDir)\src\Mono.Android\Test\Mono.Android-Tests.csproj" />
     <_ApkTestProject Include="$(_TopDir)\tests\CodeGen-Binding\Xamarin.Android.JcwGen-Tests\Xamarin.Android.JcwGen-Tests.csproj" />
     <_ApkTestProject Include="$(_TopDir)\tests\locales\Xamarin.Android.Locale-Tests\Xamarin.Android.Locale-Tests.csproj" />
-    <_ApkTestProject Include="$(_TopDir)\tests\Xamarin.Android.Bcl-Tests\Xamarin.Android.Bcl-Tests.csproj" />
+    <_ApkTestProject Include="$(_TopDir)\tests\Xamarin.Android.Bcl-Tests\Xamarin.Android.Bcl-Tests\Xamarin.Android.Bcl-Tests.csproj" />
```

Comment:
This looks weird, in that `Xamarin.Android.Bcl-Tests\Xamarin.Android.Bcl-Tests` is duplicative, and `Xamarin.Android.Bcl-Tests\Xamarin.Android.Bcl-Tests\Xamarin.Android.Bcl-Tests.csproj` is triplicative.